html/comparison.html.haml

```diff
@@ -296,8 +293,7 @@
                 %span.summary-part{"ng-show" => "diff.unmanaged_files.only_in2"}
                   {{diff.meta.description_b}}: {{diff.unmanaged_files.only_in2.files.length || 0}} files
                 %span.summary-part{"ng-show" => "diff.unmanaged_files.common"}
-                  %a{:class=>"show-common-elements", href: "#unmanaged_files_both"}
-                    both
+                  %a{:class=>"show-common-elements", href: "#unmanaged_files_both"}both
```

Comment:
Line is too long. [87/80]<br>Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace